migration/src/jira_util.py

```diff
@@ -236,6 +237,31 @@ def extract_pull_requests(o: dict) -> list[str]:
 REGEX_GITHUB_ISSUE_LINK = re.compile(r"(\s)(#\d+)(\s)")
 
 
+# common file extensions in Lucene Jira attachments
+FILE_EXT_TO_LANG = {
```

Comment:
Those extensions appear at least three times in all attachments.

Reply:
Thanks for commenting on GitHub, however, the comment would be valuable as a regular code comment 😉

Reply:
Ah yes, will do.